### PR TITLE
Redesign events feed + refactor styles

### PR DESCRIPTION
**Pull request by chancestrickland**
Note: you will likely hit a build error due to #12. I did not include my changes in `plugins/gatsby-source-eventbrite/gatsby-node.js` in the PR because I'm not convinced it's the best solution.
 
Proposed changes in this PR include:

* Manipulated the event data to organize and group by date. This could come in handy if we want to different view components (e.g., a calendar view)
* Re-designed the EventPreview component (see screenshot below) and added dummy data that should only appear in dev-mode (helps when there are no events posted in EB).
* Added the `emotion-theming` lib and restructured styles a bit to separate theme variables, mixins, and functions.
* Broke up some components

image

### CODE REVIEW

File: src/components/EventPreview.js [hunk 1]
```diff
@@ -1,35 +1,258 @@
 import React from 'react';
-import ExternalLink from '../components/ExternalLink';
-import { thumbnail, footerLinks } from './styles';
-import ImageLoader from "./ImageLoader";
+import PropTypes from 'prop-types';
+import moment from 'moment-timezone';
```
Comment (kwelch) [inline src/components/EventPreview.js]: I would prefer [`tiny-time`](https://github.com/aweary/tinytime), if possible

**Comment by kwelch** [comment]
Awesome this looks great! Thanks. I will take a look at the GraphQL stuff and then merge.

File: src/components/EventPreview.js (hunk 1 above)
Comment (chancestrickland) [reply src/components/EventPreview.js]: Sweet, shouldn't take me long to swap that out. Thanks!

**Comment by kwelch** [comment]
I have resolved the plugin issues and null venue.

File: src/components/EventPreview.js (hunk 1 above)
Comment (chancestrickland) [reply src/components/EventPreview.js]: @kwelch Updated 893227b

File: src/components/EventsList.js [hunk 2]
```diff
@@ -44,18 +44,22 @@ function EventsList({ eventGroups, theme, ...props }) {
...
-              </li>
-            ))}
-          </ul>
-        </li>
-      ))}
+      {eventGroups.map(({ day, events }) => {
+        return (
+          <li key={day} css={[styles.listItem]}>
+            <h3 css={styles.dateHeader}>
+              {tinytime(`{MMMM} {DD}, {YYYY}`).render(new Date(day))}
```
Comment (kwelch) [inline src/components/EventsList.js]: For perf reasons we should move this out of the render.

**Pull request merged**